docs/getting-started/install-elastic-endpoint.asciidoc

```diff
@@ -0,0 +1,100 @@
+[[deploy-elastic-endpoint]]
+= Install {elastic-endpoint} without Mobile Device Management
+
+To properly install and configure {elastic-endpoint} without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint} can be fully functional.
```

Comment:
This page doesn't describe installing with MDM, it describes installing manually.